packages/core/test/state.test.ts

```diff
@@ -574,4 +581,213 @@ describe('State', () => {
       expect(machine.transition(undefined, 'TO_FINAL').done).toBeTruthy();
     });
   });
+
+  describe('.can', () => {
+    it('should return true for a simple event that results in a transition to a different state', () => {
+      const machine = createMachine({
```

Comment:
Should we add an example for when only the context changes?

Reply:
Added!